glib/src/exit_code.rs

```diff
@@ -0,0 +1,22 @@
+use std::process::Termination;
+
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub struct ExitCode(i32);
```

Comment:
Maybe make the field public too, and `PartialOrd` / `Ord`, and maybe success/failure constants like in `std`?

Reply:
I think this is the only one missing?

Reply:
> Maybe make the field public too

since we have From in both directions I am not supper happy about this, but we could have `pub fn value(&self) ->i32`.

Reply:
I mean the source    at rust `pub const FAILURE: ExitCode = ExitCode(imp::ExitCode::FAILURE); ` is super obscure and I couldn't find where it comes from.

Reply:
Ok, just a `pub fn value(&self) -> i32` then :)

Reply:
And `FAILURE = 1`